src/Fabulous.Avalonia/Views/_StyledElement.fs

```diff
@@ -180,6 +184,13 @@ type StyledElementModifiers =
     static member inline styles(this: WidgetBuilder<'msg, #IFabStyledElement>, value: string list) =
         this.AddScalar(StyledElement.Styles.WithValue(value))
 
+    /// <summary>Adds inline styles used by the widget and its descendants.</summary>
+    /// <param name="this">Current widget.</param>
+    /// <param name="styles">Inline styles to be used for the widget and its descendants.</param>
+    [<Extension>]
+    static member inline inlineStyles(this: WidgetBuilder<'msg, #IFabStyledElement>, [<ParamArray>] styles: IStyle[]) =
```

Comment:
I now in a previous PR we merged the use of `[<ParamArray>]` but in a second thought I think we should be consistent and use `IStyle list` instead otherwise will be confusing .

Reply:
Also Im not convinced with current naming convention for styling.  Based on [docs](https://docs.avaloniaui.net/docs/basics/user-interface/styling/styles).

> Styles are placed inside a Styles collection element on a Control or on the Application

I think we could use the name `styles` for "inlineStyles" and rename the existing `styles` modifier to be `styleInclude`(as this essentially loads a `xaml` file) this would make it more clear IMO

Reply:
> I think we could use the name `styles` for "inlineStyles" and rename the existing `styles` modifier to be `styleInclude`(as this essentially loads a `xaml` file) this would make it more clear IMO

I agree, that would be more clear.

Reply:
> I now in a previous PR we merged the use of `[<ParamArray>]` but in a second thought I think we should be consistent and use `IStyle list` instead otherwise will be confusing .

I used a `[<ParamArray>]` to support a leaner syntax, i.e. adding styles without being forced to wrap them in a list `[style]` or array `[|style|]`. If you don't care about that in the Fabulous API, go ahead and use a `list`.